README.md

```diff
@@ -75,11 +75,11 @@ You could also use one of the sheet views from the commons module.
 ```java
 bottomSheet.showWithSheetView(new IntentPickerSheetView(this, shareIntent, "Share with...", new IntentPickerSheetView.OnIntentPickedListener() {
 	@Override
-	public void onIntentPicked(Intent intent) {
-		bottomSheet.dismissSheet();
-		startActivity(intent);
+	public void onIntentPicked(IntentPickerSheetView.ActivityInfo activityInfo) {
+        bottomSheet.dismissSheet();
```

Comment:
Indentation changed from tabs to spaces here, but I don't think that's really important here.